src/client/app/heroic/heroes/hero-list.component.ts

```diff
@@ -121,8 +122,16 @@ export class HeroListComponent implements OnDestroy, OnInit {
     this.selectedHero = hero;
   }
 
-  save(arg: { mode: 'add' | 'update'; hero: Hero }) {
-    this.heroDispatchers.save(arg.hero, arg.mode);
+  update(hero: Hero) {
+    this.save('update', hero);
+  }
+
+  add(hero: Hero) {
+    this.save('add', hero);
+  }
+
+  save(mode: 'add' | 'update', hero: Hero) {
```

Comment:
Might as well just use the service's  `add` and `update` methods directly.

Reply:
yep

Reply:
iterative